lib/raidenclient/RaidenClient.ts

```diff
@@ -251,20 +251,29 @@ class RaidenClient extends SwapClient {
     let channels: number | undefined;
     let address: string | undefined;
     let error: string | undefined;
-    const version = ''; // Intentionally left blank until Raiden API exposes it
+    let status: SwapClientStatus = SwapClientStatus.Error;
+    const version = 'v0.100.3 - Rosemary'; // Intentionally left hardcoded until Raiden API exposes it
```

Comment:
We have to remember to update this version regularly if we want to hardcode it.

Reply:
Let's never hardcode anything unless there is no other way. I checked the Raiden Client endpoints and indeed I couldn't find one returning the version. BUT each raiden log starts by printing the version in line 0: `Welcome to Raiden, version 0.100.5a1.dev157+geb2af878d!`

Reply:
I couldn't find the mentioned version description in any response of raiden API, did you get it from somewhere else @kilrau ?

Reply:
I think @kilrau means that the first row of the **log** of Raiden shows the version number. But reading that isn't a viable long term solution. I would just leave the Raiden version blank or set it to `v1` (because we are using the version 1 of their REST API)

Reply:
Agree, that it is not a long term solution but would want to use it, as it's the only way we have right now. I'll open an issue on raiden side to have something similar to `getinfo` with version and so on. Once that's there, we can switch. Deal?

Reply:
@rsercano we just agreed on the call to implement a Raiden endpoint that shows the version. Because the swaps will only be working on our Raiden fork we can easily and quickly merge that new endpoint in there. I will implement it and ping you again once it is ready.

Reply:
Great, thanks @michael1011 :)

Reply:
@rsercano it was easier than expected: https://github.com/ExchangeUnion/raiden/commit/2ac24318407084866bd81c0f212ae63a2cbfab33
![image](https://user-images.githubusercontent.com/8138838/64545932-fc359280-d329-11e9-907d-cedc1e982188.png)